README.md

```diff
@@ -114,12 +114,15 @@ Base url : `/api/wfm/[group|user|membership|`
 
 Base url : `/api/wfm/user`
 
-| resource | method | returns |
+| resource | method | returns | description |
 | -------- | ------ | ------- |
-| /auth | all | `{satus: 'ok', userId: username, sessionToken: sessiontoken, authResponse: profileData}` |
-| /verifysession | all | `{isValid: true}` |
-| /revokesession | all | `{}` |
+| /auth | all | `{status: 'ok', userId: username, sessionToken: sessiontoken, authResponse: authResponse}` | `sessionToken` : identifier for a specific user for the duration of that user's visit. <br>  `authResponse` : authentication response containing the authenticated users details. |
+| /verifysession | all | `{isValid: true}` | |
+| /revokesession | all | `{}` |  | |
+
+*Note*: You can choose which fields to remove from the authentication response by specifying the field names in the `authResponseExclusionList` in `lib/user/config-user.js`.
 
+For Example, setting the array as `['password', 'banner']` will prevent these fields from appearing in the authentication response. By default, the `password` field is removed from the response. To allow all fields to be sent, set `authResponseExclusionList` as an empty array.
```

Comment:
You also need to update the "Setup mbaas service" section for the exclusion list. This text section should be there.